### PR TITLE
General updates and reccomendations for getting started docs

### PR DESCRIPTION
**Pull request by ceejimus**
## Feature or Problem
@brooksmtownsend asked me take a gander through the docs. It's still a WIP, but this draft PR should help get the ball rolling of getting changes to prod.

### Manual Verification
I used the magic of `npm` to run the site locally :)

### CODE REVIEW

File: docs/fundamentals/actors/create-actor/test.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import TabItem from "@theme/TabItem";
 
 Now that your actor project is generated we can write a simple unit test to verify its functionality. We want to ensure that our logic to parse the query string for a name and return `Hello <name>` is correct.
 
-Right now, our actor is laid out in a way that is difficult to test properly. Let's look at the function signature inside of the `impl` block:
+Let's revert our actor to the simpler initial version. Our actor is laid out in a way that is difficult to test properly. Let's look at the function signature inside of the `impl` block:
```
Comment (brooksmtownsend) [inline docs/fundamentals/actors/create-actor/test.mdx:13]: I don't think we're reverting here but I do like the idea of rewording this.
```suggestion
Our actor code is currently organized in a way that is difficult to test properly. Let's look at the function signature inside of the `impl` block:
```

File: docs/fundamentals/actors/index.md [hunk 2]
```diff
@@ -7,30 +7,45 @@ sidebar_position: 0
...
+[actor model](https://en.wikipedia.org/wiki/Actor_model)
+and have experience implementing actors in other projects or tools,
+feel free to [skip to the next section](./wc_actors).
+
+The actor model is a model of concurrent computation that's been around since
+**1973** and is as meaningful and applicable today than it was 50 years ago.
+In this model, the **actor** is the primitive building block
+of concurrent computation.
+Actors are "black-box" computational entities that can only communicate with
+other actors by passing messages and creating new actors.
```
Comment (brooksmtownsend) [inline docs/fundamentals/actors/index.md:20]: Just to confirm, is this line break change intended to lay these out on different lines or is it just editor word wrap configuration? Word wrap is fine, markdown ignores single line breaks

File: docs/installation.mdx [hunk 3]
```diff
@@ -110,4 +111,13 @@ docker logs -f wasmcloud
...
+Verify that `wash` is properly installed with:
+
+```bash
+wash --help
+```
+
+If wash is installed correctly, you will see a printout of all available commands and short descriptions for each.
+If you ever need more detail on a specific command or sub-command just run `wash <command> --help`.
+
+Now that `wash` is installed, let's [get started](/docs/tour/hello_world).
```
Comment (brooksmtownsend) [inline docs/installation.mdx:123]: Love this extra validation note

File: docs/intro.md [hunk 4]
```diff
@@ -12,12 +12,11 @@ Get started building applications with wasmCloud today and enjoy the freedom of
 
 wasmCloud is designed around the following core tenets:
 
-- Dead simple Distributed Applications
+- Dead simple distributed applications
 - Run anywhere
 - Secure by default
-- Save time, money, and effort
```
Comment (brooksmtownsend) [inline docs/intro.md:18]: @autodidaddict any big reservations on removing this one? I don't have an issue with focusing on the other traits

File: docs/tour/hello_world.mdx [hunk 5]
```diff
@@ -29,18 +29,24 @@ use `wash` to start one now.
 
 ## Starting a Host
 
-To start a host, simply run `wash up`. This will start a host with default configurations which is perfect for this tutorial, and you can exit at any time with `CTRL+c`
+To start a host, simply run `wash up`. The default settings should work well for this tutorial. The host can be killed at any time with `CTRL+c`
 
-By default, `wash` will run the host in interactive mode. If you'd rather run the host in the background, run `wash up --detached` (or `wash up -d`, for short).
+This command does a lot behind the scenes, but most importantly it spins up 3 core processes:
+- the **wasmCloud** host which manages the execution of actors and capability providers
+- a **NATS** server to manage communications b/w application components
```
Comment (brooksmtownsend) [inline docs/tour/hello_world.mdx]:
```suggestion
- a [**NATS**](https://nats.io/) server to manage communications between application components
```

File: docs/tour/hello_world.mdx [hunk 6]
```diff
@@ -29,18 +29,24 @@ use `wash` to start one now.
...
+- a **wadm** process which monitors the [lattice](/docs/reference/glossary#lattice) and maintains the state of managed deployments
```
Comment (brooksmtownsend) [inline docs/tour/hello_world.mdx]:
```suggestion
- a [**wadm**](/docs/category/managed-application-deployment-wadm) process which monitors the [lattice](/docs/reference/glossary#lattice) and maintains the state of managed deployments
```

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]
Just a couple of small requests, and you'll just have to sign your commits off to meet the DCO (should be just a single git rebase command with `--signoff` at the end!)

These are all really awesome fixes and I think we should merge this PR as soon as the DCO is fixed / comments addressed, and any other changes can go in subsequent PRs 😄

File: docs/fundamentals/actors/create-actor/test.mdx (hunk 1 above)
Comment (ceejimus) [reply docs/fundamentals/actors/create-actor/test.mdx:13]: > I don't think we're reverting here but I do like the idea of rewording this.

The reason I chose that specific working is that, in the previous section, the actor code is modified to include another query param - "msg". But the code in this section is that of the original single param functionality. Could move the msg code into this section and modify the test?

File: docs/fundamentals/actors/index.md (hunk 2 above)
Comment (ceejimus) [reply docs/fundamentals/actors/index.md:20]: I don't have auto wrap config'd, but I like to break up lines for readability. I did not intend for these to be on separate rendered lines. The docs sometimes break up lines and sometimes don't, so I figured it was fairly harmless.

File: docs/installation.mdx (hunk 3 above)
Comment (ceejimus) [reply docs/installation.mdx:123]: Linux PATH b like that sometimes xD

File: docs/fundamentals/actors/create-actor/test.mdx (hunk 1 above)
Comment (brooksmtownsend) [reply docs/fundamentals/actors/create-actor/test.mdx:13]: Ah, I see. This feels like a good way to phrase this section then and we can revisit later (really, we just need a bit of a better testing strategy!)

**Review by brooksmtownsend — APPROVED** [review]
Love it!

File: docs/tour/hello_world.mdx [hunk 7]
```diff
@@ -29,18 +29,24 @@ use `wash` to start one now.
...
 ## Generating a New Project
 
-Now that we have a host to run our application on, let's generate a new actor project. This is where our "Hello World" example begins. If you're running your host in the foreground, just switch to another terminal window to run these commands.
+Now that our wasmCloud host is running, let's generate a new actor project. This is where our "Hello World" example begins. If you're running your host in the foreground, just switch to another terminal window to run these commands.
 
 This command generates a new actor project in the `./hello` directory:
 
 ```shell
-wash new actor hello
+wash new actor -t hello hello
```
Comment (mfreeman451) [inline docs/tour/hello_world.mdx:49]: Can we do a better job explaining what -t (template) is for here?

Comment (brooksmtownsend) [reply docs/tour/hello_world.mdx:49]: Good idea, we could just go back and add the long argument for clarity

**Pull request merged**